micromamba/tests/test_env.py

```diff
@@ -124,3 +122,19 @@ def test_env_export(self):
         assert ret["name"] == env_name
         assert set(ret["channels"]) == {"https://conda.anaconda.org/conda-forge"}
         assert "micromamba=0.24.0=0" in ret["dependencies"]
+
+    def test_env_remove(self):
+        env_name = "env-create-remove"
+        # Create env with xtensor
+        res = create("xtensor", "-n", env_name, "--json", no_dry_run=True)
```

Comment:
I don't see any tests with the environments file here. I think it would be nice to also check that the environment is mentioned in `~/.conda/environments.txt` file